src/backend/distributed/utils/maintenanced.c

```diff
@@ -214,9 +214,12 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 {
 	Oid databaseOid = DatumGetObjectId(main_arg);
 	MaintenanceDaemonDBData *myDbData = NULL;
+	ErrorContextCallback errorCallback;
+#ifdef HAVE_LIBCURL
```

Comment:
This is kind of annoying just for variable declarations. Does it really make sense to define these out rather than just leave the (hopefully low-impact) `difftime` call in-place?

If not, can you investigate whether the appearance of this could be improved by adding a `USED_WITH_LIBCURL_ONLY` macro similar to `PG_USED_FOR_ASSERTS_ONLY` such that when `HAVE_LIBCURL` is false it expands to `pg_attribute_unused`?